### PR TITLE
Add notes on launching from VS Code

### LINKED ISSUE #263: document starting devtools from vs code
aka, filling in text for this file: https://github.com/flutter/devtools/blob/master/docs/vscode.md

I suspect this could be pretty brief, and just point people to the relevant Dart-Code docs.

@DanTup, do you mind taking this?

### PR DESCRIPTION
**Pull request by DanTup**
I'm not sure if we wanted screenshots, but I already had them so added them in.

Something like this will go into the release notes (and onto dartcode.org) but given it's fairly simple I thought it may be better to have here inline rather than send the user off to different site, but lmk if you'd prefer otherwise.

(Fixes #263).

### CODE REVIEW

**Comment by DanTup** [comment]
(@devoncarew)

File: docs/vscode.md [hunk 1]
```diff
@@ -2,4 +2,33 @@
...
+
+## Start an application to debug
+
+Start a debug session for your application by opening the root folder of your project (the one containing `pubspec.yaml`) in VS Code and clicking **Debug > Start Debugging** (`F5`).
+
+## Launch the DevTools
+
+Once the debug session is active and the application has started, the **Dart: Open DevTools** command will become available in the VS Code command palette:
+
+<img src="{{ "/assets/images/vscode/command.png" | relative_url }}" width="700" height="115" />
```
Comment (devoncarew) [inline docs/vscode.md]: Is this pattern necessary for all url references?

```
{{ "/assets/images/vscode/command.png" | relative_url }}
```

It is a bit verbose, and the nice thing about markdown docs are that they're generally simple to author.

**Comment by DanTup** [comment]
> One thought re: the images paths; they're currently `assets/images/vscode/command.png`; you may consider removing a few directories to make references shorter; `images/vscode/command.png`? `vscode/command.png`?

I saw the assets folder had a css folder inside it, so figured it might be for images too - I can move it up if you're ok with that (I could also drop vscode and/or make it a prefix to the filename?)

File: docs/vscode.md (hunk 1 above)
Comment (DanTup) [reply docs/vscode.md]: Define necessary :)

If the user can get to the page both with and without a trailing slash, I think we need absolute or root-relative paths. It doesn't seem possible on live in my testing, but does with `jekyll serve`, though I'm not sure why - on my blog everything ends up with trailing slashes, and it seems like [this has changed](http://marcgg.com/blog/2016/03/14/github-pages-jekyll/) unexpectedly in the past.

We could removing `| relative_url` and manually put a root-relative path, but it means hard-coding the `/devtools/` prefix and I don't know if we'd want that (we could also use `{{ site.base }}` but that just seems very similar to what we have).

LMK what you think would be best. If we think the trailing slashes will never work, I can just put relative paths in directly - I often default to over-cautious =)

**Comment by devoncarew** [comment]
`//docs/images/vscode-foo2.png` or similar sounds good. I only used the assets folder for the css file since that played nicer w/ jekyll.

File: docs/vscode.md (hunk 1 above)
Comment (devoncarew) [reply docs/vscode.md]: > but it means hard-coding the `/devtools/` prefix

That sounds reasonable; you have more jekyll experience than me however, so I'll defer to whichever you think best.

**Comment by DanTup** [comment]
Pushed 905a527fa9e54fa3768aeace23ce2492c7e28086 which moves the files up (`images` out of `assets` and uses `vscode_` prefix instead of a folder) and also removes the tags around the images (this should be fine as long as we don't somehow end up with trailing slashes on the page urls).

**Comment by DanTup** [comment]
Possibly flaky tests:

```
01:26 +10: test/integration_tests/integration_test.dart: integration debugging break on exceptions
chrome • console:log • [{"id":1}]
chrome • console:log • [{"id":2,"result":"debugger"}]
chrome • console:log • [{"id":3,"result":"running"}]
chrome • console:log • [{"id":4}]
chrome • console:log • [{"id":5,"result":"running"}]
chrome • console:log • [{"id":6,"result":"paused"}]
chrome • console:log • [{"id":7,"result":"file:///Users/travis/build/flutter/devtools/packages/devtools/test/fixtures/debugging_app_exception.dart:25"}]
chrome • console:log • [{"id":8,"result":["<exception>:StateError","foo:2"]}]
chrome • console:log • [{"id":9}]
chrome • console:log • [{"id":10}]
chrome • console:log • [{"id":11,"result":"paused"}]
01:30 +10 -1: test/integration_tests/integration_test.dart: integration debugging break on exceptions [E]
  Expected: 'running'
    Actual: 'paused'
     Which: is different.
            Expected: running
              Actual: paused
                      ^
             Differ at offset 0
  
  package:test_api                            expect
  test/integration_tests/debugger.dart 225:5  debuggingTests.<fn>
  ===== asynchronous gap ===========================
  dart:async                                  _AsyncAwaitCompleter.completeError
  test/integration_tests/debugger.dart        debuggingTests.<fn>
  ===== asynchronous gap ===========================
  dart:async                                  _asyncThenWrapperHelper
  test/integration_tests/debugger.dart        debuggingTests.<fn>
```

I'll re-run since I'm sure it's unrelated to this and keep my eye open to see if it comes up again.

**Pull request merged**